.github/workflows/ruby.yml

```diff
@@ -0,0 +1,61 @@
+name: Ruby
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: yes
+          MYSQL_DATABASE: polymorpheus_test
+        ports:
+          - 3306:3306
+        options: >-
+          --health-cmd="mysqladmin ping"
+          --health-interval=10s
+          --health-timeout=5s
+          --health-retries=3
+
+    strategy:
+      fail-fast: false
+      matrix:
+        gemfile: ['5.2', '6.0']
+        ruby-version: ['2.6', '2.7', '3.0']
```

Comment:
Is there a benefit to including 3.0 if it's excluded from the matrix for both rails versions?

Reply:
@jchuang Hmm, it looks like I can remove that exclusion from the list. I think I excluded ruby 3.0 / rails 6.0 originally because of [this chart](https://www.fastruby.io/blog/ruby/rails/versions/compatibility-table.html), but it's working for me now. I don't mind testing against all theoretically supported, maintained versions.